tests/test-backend-ops.cpp

```diff
@@ -2225,7 +2225,11 @@ int main(int argc, char ** argv) {
             continue;
         }
 
+#ifdef GGML_USE_QNN
+        ggml_backend_t backend = ggml_backend_reg_init_backend(i, "/data/local/tmp/");
+#else
```

Comment:
Generally, it is not ok to add backend-specific code to `test-backend-ops`. Instead, you can make `"/data/local/tmp/"` the default value when it is `NULL` in `ggml_backend_qnn_reg_init`.

Reply:
modify code according to review comments done.

thanks much for your review comments. it's really useful.